src/test/java/seedu/address/model/tuition/TuitionTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.tuition;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
+import static seedu.address.testutil.TypicalStudents.ALICE;
+import static seedu.address.testutil.TypicalStudents.BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.session.Session;
+import seedu.address.model.student.Student;
+import seedu.address.testutil.SessionBuilder;
+import seedu.address.testutil.StudentBuilder;
+
+class TuitionTest {
+
+    private static final Session exampleSession = new SessionBuilder().build();
+
+    @Test
+    void getStudent() {
+        Tuition aliceInTuition = new Tuition(ALICE, exampleSession, 0 , 0);
+        assertEquals(aliceInTuition.getStudent(), ALICE);
+
+        assertNotEquals(aliceInTuition.getStudent(), BOB);
+
+        assertNotEquals(aliceInTuition.getStudent(), null);
+    }
+
+    @Test
+    void getSession() {
+        Tuition bobInTuition = new Tuition(BOB, exampleSession, 0, 0);
+        assertEquals(bobInTuition.getSession(), exampleSession);
+
+        Session tenMinuteSession = new SessionBuilder().withDuration("10").build();
+        assertNotEquals(bobInTuition.getSession(), tenMinuteSession);
+
+        assertNotEquals(bobInTuition.getSession(), null);
+    }
+
+    @Test
+    void getSessionIndex() {
+        Tuition aliceInTuition = new Tuition(ALICE, exampleSession, 1, 2);
+        assertEquals(aliceInTuition.getSessionIndex(), 3);
+
+        assertNotEquals(aliceInTuition.getSessionIndex(), 2);
```

Comment:
Actually I think this part is abit confusing for me, aliceInTuition when calling new Tuition() has a sessionIndex of 2, however, we assertNotEquals that getSessionIndex() is 2, isit because of the difference between the zero index and one index?

Reply:
Yes correct, this is because of the `OneBasedIndex` that is returned in `getSessionIndex()`. I will add in a comment to make this clearer!